From 3ac455c18b77d419fc5b569b682feb2cca1bcf15 Mon Sep 17 00:00:00 2001
From: Farzin Kazemzadeh <itisFarzin@proton.me>
Date: Wed, 10 Dec 2025 08:58:59 +0000
Subject: [PATCH 4/4] Enable UDFPS animations

Signed-off-by: Farzin Kazemzadeh <itisFarzin@proton.me>
---
 voltage_gsi.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/voltage_gsi.mk b/voltage_gsi.mk
index 9399786..18ab0df 100644
--- a/voltage_gsi.mk
+++ b/voltage_gsi.mk
@@ -16,6 +16,9 @@ WITH_ADB_INSECURE := true
 # GSIs don't have kernel
 TARGET_NO_KERNEL := true
 
+# UDFPS animations
+EXTRA_UDFPS_ANIMATIONS := true
+
 # Enable face unlock
 TARGET_FACE_UNLOCK_SUPPORTED := true
 
-- 
2.43.0

